Driverless AI/machine-learning-experiment-scoring-and-analysis-financial-focus/machine-learning-experiment-scoring-and-analysis-financial-focus.md

```diff
@@ -990,42 +727,39 @@ Below are three questions to test your understanding of the experiment summary a
 	
 **Note:** If you are not sure what Log loss is, feel free to review the concepts section of this tutorial.
 
-
-3\. So what do the Log Loss values tell us?  The essential Log Loss value is the test score value. This value tells us how well the model generated did against the freddie_mac_500_test set based on the error rate. In case of experiment **Freddie Mac Classification Tutorial**, the **test score LogLoss = 0.1191906 +/- 0.00122753** which is the log of the misclassification rate. The greater the Log loss value the more significant the misclassification. For this experiment, the Log Loss was relatively small meaning the error rate for misclassification was not as substantial. But what would a score like this mean for an institution like Freddie Mac?
+3\. So what do the Log Loss values tell us?  The essential Log Loss value is the test score value. This value tells us how well the model generated did against the freddie_mac_500_test set based on the error rate. In case of experiment **Freddie Mac Classification Tutorial**, the **test score LogLoss = 0.119566 +/- 0** which is the log of the misclassification rate. The greater the Log loss value the more significant the misclassification. For this experiment, the Log Loss was relatively small meaning the error rate for misclassification was not as substantial. But what would a score like this mean for an institution like Freddie Mac?
```

Comment:
rewrite this: This value tells us how well the model generated did against the freddie_mac_500_test set based on the error rate.

to this: This value tells us how well the generated model did against the freddie_mac_500_test set based on the error rate.